libeos-updater-flatpak-installer/perform-flatpak-actions.c

```diff
@@ -69,6 +69,16 @@ try_update_application (FlatpakInstallation       *installation,
           return TRUE;
         }
 
+     /* We also have to check for FLATPAK_ERROR_ALREADY_INSTALLED since this
+      * is thrown if there are no updates to complete. Arguably a design flaw
+      * in Flatpak itself. */
```

Comment:
We probably can’t change the behaviour of libflatpak to no longer return this error on calls to `flatpak_installation_update()`, but we could definitely make sure it’s mentioned in the documentation for that function.

Reply:
https://github.com/flatpak/flatpak/pull/1337/files